src/features/lightspeed/playbookGeneration.ts

```diff
@@ -238,6 +238,25 @@ export async function showPlaybookGenerationPage(extensionUri: vscode.Uri) {
         panel.dispose();
         break;
       }
+      case "resetOutline": {
+        vscode.window
```

Comment:
You can't show a modal dialog from the _view_ so we need to send an event from the _view_ to the _controller_.

The _controller_ then shows the dialog and, if the user confirms, sends another event to reset the Outline.